docs/competition/teams/preparation.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Please could we keep this page at its original name? (`competition-preparation.md`)

Aside from making the diff easier to read, it's not especially clear what "preparation" this document currently refers to.

Reply:
Ah, I had imagined this staying under `teams`, rather than `competition/teams` (I hadn't spotted it had also moved).

Reply:
I've not looked around for other similar information, though my gut feel is that this probably belongs in the teams section more than the competition section (I think mostly because it's not really new information about the competition?). I don't know what's best though.

I guess a comparison would be where similar information related to a tech day or kickstart would be and what the other information in this `competition/teams` section is.

Up to you :man_shrugging:.

Reply:
My intention is to go through the competition section after this and tidy things up a bit.

My thoughts around the `competition/` section was to show things related to the competition itself, which IMO this is. I guess `competition/teams` is the union of `teams/` and `competition/`.

Reply:
So I see the "competition" section as being mostly about the Competition event itself. I do think we perhaps need a clearer way to group things which are about the more general competition year though.

This is mostly based on the way the existing docs were structured though.